src/views/conference/2019/index/l10n.json

```diff
@@ -23,9 +23,10 @@
     "conference-2019.kenyaTitle": "Scratch2019NBO",
     "conference-2019.kenyaSubTitle": "Waves of Innovation",
     "conference-2019.kenyaDesc": "In recognition of Africa's technological contributions to the world and the potential of the youth of Africa, Scratch2019NBO will be held in Nairobi, Kenya. Join educators from around the world to share lessons, empower young people, and celebrate accomplishments in creative coding.",
+    "conference-2019.kenyaPostpone": "The Scratch2019NBO conference, originally planned for Nairobi, Kenya in July 2019, has been postponed. Information about future plans will be available later this year.",
```

Comment:
This will have the same missing string id because it's new, and it won't be in scratch-l10n.